OracleLinuxDevelopers/oraclelinux8/redis/5/Dockerfile

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2023 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+FROM ghcr.io/oracle/oraclelinux:8
+
+RUN dnf install -y redis && \
+    rm -rf /var/cache/dnf
```

```suggestion
RUN dnf -y module enable redis:5 && \
    dnf -y install redis && \
    rm -rf /var/cache/dnf
```

This specifically enables the `redis:5` AppStream instead of relying on defaults which have an annoying tendency of sometimes not being what you expect them to be.